Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/outbound/deferred/request/StandardOutboundDocSubmissionDeferredRequestTest.java

```diff
@@ -67,61 +70,99 @@
  */
 public class StandardOutboundDocSubmissionDeferredRequestTest {
 
-    protected Mockery context = new JUnit4Mockery() {
-        {
-            setImposteriser(ClassImposteriser.INSTANCE);
-        }
-    };
-    final XDRAuditLogger mockXDRLog = context.mock(XDRAuditLogger.class);
-    final XDRPolicyChecker mockPolicyCheck = context.mock(XDRPolicyChecker.class);
-    final SubjectHelper mockSubjectHelper = context.mock(SubjectHelper.class);
-    final OutboundDocSubmissionDeferredRequestDelegate mockDelegate = context.mock(OutboundDocSubmissionDeferredRequestDelegate.class);
+    private final String senderHCID = "1.1";
+    private final String receiverHCID = "2.2";
+    private final DocSubmissionDeferredRequestAuditLogger mockAuditLogger
+        = mock(DocSubmissionDeferredRequestAuditLogger.class);
+    private final XDRPolicyChecker mockPolicyChecker = mock(XDRPolicyChecker.class);
+    private final SubjectHelper mockSubjectHelper = mock(SubjectHelper.class);
+    private final OutboundDocSubmissionDeferredRequestDelegate mockDelegate
+        = mock(OutboundDocSubmissionDeferredRequestDelegate.class);
 
     @Test
     public void testProvideAndRegisterDocumentSetB() {
-        expect2MockAudits();
-        setMockPolicyCheck(true);
-        setMockSubjectHelperToReturnValidHcid();
-        setMockDelegateToReturnValidResponse();
+        NhinTargetCommunitiesType targets = createNhinTargetCommunitiesType(receiverHCID);
+        ProvideAndRegisterDocumentSetRequestType request = new ProvideAndRegisterDocumentSetRequestType();
+        final AssertionType assertion = createAssertion(senderHCID);
+        UrlInfoType urlInfo = new UrlInfoType();
 
-        XDRAcknowledgementType response = runProvideAndRegisterDocumentSetBAsyncRequest();
+        when(mockPolicyChecker.checkXDRRequestPolicy(any(ProvideAndRegisterDocumentSetRequestType.class),
```

Comment:
This when statement is necessary in testProvideAndRegisterDocumentSetB because we wanted to return a true when the policyCheck is performed in the actual class.  The StandardOutboundDocSumbissionDeferredRequest class calls isPolicyValid which in-turns call policyChecker.checkXDRRequestPolicy and we want this method to return true to get a successful response.